routes.js

```diff
@@ -0,0 +1,5 @@
+const routes = {
+    '/user/settings': '/user_settings',
```

Comment:
I would consider adding the complete service path, so we can replace it easier in the app.
`/users-service/user/settings`
What do you think?

Reply:
done